### PR TITLE
implementing firebase firestore in my project

### PR DESCRIPTION
**Pull request by Mucyo-Aime-Christian**
#firebase firestore in my projects
- firebase configuration
- forms validation
- storing in database 
- sending comments and retrieving them from database
- validating contact page
- validating sign in page
- validating comments page
- validating subscription panel

### CODE REVIEW

**Comment by James-nyagatare** [comment]
Everything looks good for me but you should display queries in the admin panel.

File: UI/blog.html [hunk 1]
```diff
@@ -43,31 +43,31 @@ <h3><a href="index.html">mr chris</a></h3>
       <h3>BEACH VACATION</h3><br/>
```
Comment (harerakalex) [inline UI/blog.html:43]: @Mucyo-Aime-Christian good job, however, I have got some feedback on your PR. Never hard code something that you can style. for example, use CSS to put this heading in Uppercase.

File: UI/blog.html [hunk 2]
```diff
@@ -96,24 +96,57 @@ <h3>FEATURES</h3><br/>
...
+                <a href="https://www.twitter.com/chrisdolla17"><i class="fa fa-twitter"></i></a>
                 <a href="https://www.instagram.com/chris_.dolla"><i class="fa fa-instagram"></i></a>
             </div>
         </div>
         <div class="footer2"><span>Copyright ©2020 All rights reserved</span>| This template is made by Mr Chris</div>
     </footer>
 </main>
 </div>
+<!--validating and storing subscribers-->
+<script>
```
Comment (harerakalex) [inline UI/blog.html:119]: I was wondering if you can use a separate file for javascript and you connect to it.

File: UI/login.html [hunk 3]
```diff
@@ -13,28 +22,49 @@
...
-              <a href="#">Create your account ➡ </a>
+              <a href="signup.html">Create your account ➡ </a>
           </span>
           <span><a href="./index.html">Back</a></span>
       </div>
-  </div>
+    </form>
+
+
+    <script>
```
Comment (harerakalex) [inline UI/login.html:50]: This should be in a separate file.

**Review by harerakalex — CHANGES_REQUESTED** [review]

**Pull request closed without merging**